Utils/src/JetUncertaintyProducer.cc

```diff
@@ -71,7 +71,8 @@ void JetUncertaintyProducer::produce(edm::StreamID, edm::Event& iEvent, const ed
 		edm::Ptr<pat::Jet> jetPtr = jets->ptrAt(idx);
 		pat::Jet ajet(jetPtr);
 		math::XYZTLorentzVector vjet = ajet.p4();
-		
+		ajet.addUserInt("origIndex",idx);
```

Comment:
I'm really commenting on line 69. How does computing the idx every time, but using and iterator for the "jets" handle save any time. Isn't std::distance slow. Wouldn't it be better to simply compute the size of the jets vector once and use the for loop index? This might be inconsequential because we don't expect that many jets, but I felt like mentioning it.

Reply:
I think I copied this from somewhere in CMSSW. Might be worth making the minor optimization you suggest.

Reply:
Tested a change to this and it's not any faster. I'll commit it anyway just to simplify the code.

Reply:
Both interesting and good to know. Thanks for looking into it.